setup.py

```diff
@@ -697,8 +697,7 @@ def run(self):
 
 
 INSTALL_VERSION_TEMPLATE = '''\
-# This file was auto-generated by salt's setup on \
-{date:%A, %d %B %Y @ %H:%m:%S UTC}.
```

Comment:
Well, the [corresponding commit](https://github.com/saltstack/salt/pull/41795/commits/0c6611f274b95d42055f62786c82a9812e8c0a1d) says all by itself. We had a [registered bug in our Bugzilla](https://bugzilla.suse.com/show_bug.cgi?id=969407) for this.